templates/car_input.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <h1>Welcome to the Automotive Solution Tool!</h1>
+</head>
+<body>
+
+    <p>Please fill out the boxes below to search for your desired car:</p>
+
+    <form method="GET" action="get_car">
+        
+        <form>
+            Make: <input type="text" name="make"><br />
+            <br>
+            Model: <input type="text" name="model"><br />
+            <br>
+            Model Year: <input type="number" name="year"><br />
```

Comment:
This is the first classification for a car, it should be at the top of the inputs